api/tests/test_push_log.py

```diff
@@ -0,0 +1,81 @@
+
+def trim_log(caplog):
+    return caplog.text.replace('\n', '')
+
+def test_push_log_controller_with_valid_json_no_extra(client, caplog):
```

Comment:
praise: tests look good to me! It seems to me that we have all cases covered properly. +1 for caplog, didn't know about this fixture!

Reply:
Thank you very much!